From: Gabriel M. Beddingfield <gabriel@teuton.org>
Description: Replace using of QString(int) with QString(const char*) to fix
 FTBFS on Ubuntu Maverick (alpha).
---
 src/Tritium/src/Song.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)
--- a/src/Tritium/src/Song.cpp
+++ b/src/Tritium/src/Song.cpp
@@ -439,7 +439,7 @@ namespace Tritium
 	song->set_humanize_velocity_value( 0.0 );
 	song->set_swing_factor( 0.0 );
 
-	T<Instrument>::shared_ptr inst( new Instrument( QString(0), "New instrument", new ADSR ) );
+	T<Instrument>::shared_ptr inst( new Instrument( QString("0"), "New instrument", new ADSR ) );
 	T<Sampler>::shared_ptr sampler = engine->get_sampler();
 	sampler->clear();
 	sampler->add_instrument( inst );
